src/sciline/pipeline.py

```diff
@@ -149,7 +149,7 @@ def _find_nodes_in_paths(graph: Graph, end: Key) -> List[Key]:
 
 
 def _is_multiple_keys(
-    keys: type | Iterable[type] | Item[T],
+    keys: type | Iterable[type] | Item[T] | object,
```

Comment:
Isn't this equivalent to `keys: Any` now? Why did you add `object`?

Reply:
Yes, I could not get it to work otherwise. I thought listing the others still serves as an expression of intent?

Reply:
Maybe. So did it not work with `Any`? From what I understand that is preferred over `object`.